curp/tests/common/mod.rs

```diff
@@ -1,82 +1,3 @@
 #![allow(dead_code, unused)]
```

Comment:
Please remove this

Reply:
The integration tests contains some unused code, this lint is needed